erts/preloaded/src/erlang.erl

```diff
@@ -1210,12 +2595,36 @@ float_to_list(_Float, _Options) ->
 
 %% floor/1
 %% Shadowed by erl_bif_types: erlang:floor/1
+-doc """
+Returns the largest integer not greater than `Number`. For example:
+
+```erlang
+> floor(-10.5).
+-11
+```
+
+Allowed in guard tests.
+""".
+-doc(#{since => <<"OTP 20.0">>}).
 -spec floor(Number) -> integer() when
       Number :: number().
 floor(_) ->
     erlang:nif_error(undef).
 
 %% fun_info/2
+-doc """
+Returns information about `Fun` as specified by `Item`, in the form
+`{Item,Info}`.
+
+For any fun, `Item` can be any of the atoms `module`, `name`, `arity`, `env`, or
+`type`.
+
+For a local fun, `Item` can also be any of the atoms `index`, `new_index`,
+`new_uniq`, `uniq`, and `pid`. For an external fun, the value of any of these
+items is always the atom `undefined`.
+
+See [`erlang:fun_info/1`](`fun_info/1`).
+""".
 -spec erlang:fun_info(Fun, Item) -> {Item, Info} when
```

Comment:
btw, is there a reason this is `-spec erlang:fun_info(...)` instead of `-spec fun_info(...)`?

We thought there was an ExDoc bug when rendering these specs, adding the redundant module name.

Reply:
> Auto-imported BIFs are listed without module prefix. BIFs listed with module prefix are not auto-imported.

https://github.com/garazdawi/otp/blob/lukas/otp/convert-docs-to-markdown/erts/preloaded/src/erlang.erl#L31

It is the same in the current docs. I suppose we could change to use a metadata tag for this now with ExDoc.

Reply:
For spec-purposes, I think we should always remove the module name, as no other module includes them. Using metadata is a good idea!

You could also use the metadata and keep them separately in the sidebar, like Elixir does to separate guards in its `Kernel`.

Reply:
> You could also use the metadata and keep them separately in the sidebar, like Elixir does to separate guards in its Kernel.

[I already did that](https://erlang.github.io/prs/8026/erts-14.2.1/doc/html/erlang.html), but I've pushed a commit that also removes the `erlang:` prefix and adds the information as an annotation.